README.md

```diff
@@ -134,7 +134,7 @@ You also need to get the blockchain plugin submodule and install it.
 
 ```sh
 git submodule update --init --recursive
-poetry -C kevm-pyk run kdist --verbose build evm-semantics.plugin
+CXX=clang++-15 poetry -C kevm-pyk run kdist --verbose build evm-semantics.plugin
```

Comment:
In line 143, this is drawn attention to.

Reply:
Since `install-build-deps` installs clang-15 by default, I think it would be great to set this as the default as well.